packages/terra-search-field/CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 * Fixed
   * Removed unnecessary aria labels and roles to fix redundant announcement of search group issue.
+  * Fixed port resized 320x256 px for `terra-search-field`.
+  * Added styles for "focus me" button for `terra-search-field`.
```

Comment:
This change was made in example not the component itself
```suggestion
```

Reply:
Updated